articles/user-guide/machines/index.md

```diff
@@ -17,7 +17,7 @@ Quantum simulators are software programs that run on classical computers and act
 
 ## The Quantum Development Kit (QDK) quantum simulators
 
-The quantum simulator is responsible for providing implementations of quantum primitives for an algorithm. This includes primitive operations such as `H`, `CNOT`, and `Measure`, as well as qubit management and tracking. The QDK includes different classes of quantum simulators representing different run models for the same quantum algorithm. 
+The quantum simulator is responsible for providing implementations of quantum primitives for an algorithm. This includes primitive operations such as `H`, `CNOT`, and `Measure`, as well as qubit management and tracking. The QDK includes different classes of quantum simulators representing different execution models for the same quantum algorithm. 
```

Comment:
Also cannot use execution here.

Reply:
See https://github.com/MicrosoftDocs/quantum-docs-pr/pull/1068#discussion_r499703896.

Reply:
Opened #1069 for this, please take a look there.